src/platforms/kotlin-multiplatform/swift-package-manager.mdx

```diff
@@ -59,3 +63,8 @@ kotlin {
   }
 }
 ```
+
+## Current Limitations
+
+- SPM can only be used if the Kotlin Multiplatform framework is exported as static framework.
```

Comment:
```suggestion
- SPM can only be used if the Kotlin Multiplatform framework is exported as a static framework.
```

This seems like a prerequisite? "Kotlin Multiplatform must be exported as a static framework."

Reply:
Hm not sure where the best place to put that would be since this step is already provided through the code snippet e.g `isStatic = true`. I added it here to give it more visibility.

But yeah, maybe it's better to put something this into the prereq: 
"Make sure your Kotlin Multiplatform framework is exported statically via `isStatic = true`" or something like that

Reply:
Yeah, I agree with putting this as a prereq.